packages/client-admin/src/context/auth.context.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { createContext, FC, useContext, useEffect, useState } from 'react';
 import jwt_decode from 'jwt-decode';
 
-export interface Token {
-  token: string;
+export interface DecodedToken {
```

Comment:
For the token, I wonder if its a good idea to add the token contents as an orphaned type on the GraphQL schema so the correct type definitions can be generated and shared

Reply:
That's good idea, haven't thought about that

Reply:
Looks like you need to have a query or mutation to pull it into the client, skipping for now